mindmaps-graph/src/main/java/io/mindmaps/example/MovieGraphFactory.java

```diff
@@ -309,8 +280,10 @@ private static void buildRules() {
                 .addConclusion(person).addConclusion(genre).addHypothesis(hasCast);
     }
 
-    private static Entity putEntity(EntityType type, String name) {
-        return mindmapsGraph.putEntity(name.replaceAll(" ", "-").replaceAll("\\.", ""), type);
+    private static Entity putEntityWithName(EntityType type, String value) {
```

Comment:
I will fix that in my next PR. Don't want to run all the tests over again just for one internal refactor.